{"bug_id": "5340", "title": "Bug 5340Cancelling add exception breakpoint has no effect", "description": "1) Go to the breakpoints pane in the debug perspective2) Click the J! button to add an exception3) In the progress dialog, click cancel. The progress dialog closes immediately4) Eventually the exception list comes up, it ignored the cancelation request.It should either honour the cancellation request or diable the cancel button(when calling ProgressMonitorDialog.run pass false for the canceleableparameter)."}